src/Altinn.Platform/Altinn.Platform.Events/Tests/Data/apps/tdd/endring-av-navn/config/process/process.bpmn

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We do not need all of these files from the app in the test project. 
Please see if you can delete 
 - process.bpmn
 - appsettings.development.json
 - might not need appsettings either(?)
 -  Skjema.cs
 - all files related to ui and prefill

Reply:
Removed unused apps